performance/tests/VirtualList.test.js

```diff
@@ -1,128 +1,6 @@
-
-			const averageFPS = await getAverageFPS();
-			TestResults.addResult({component: component, type: 'FPS Mousewheel', actualValue: Math.round((averageFPS + Number.EPSILON) * 1000) / 1000});
-
-			expect(averageFPS).toBeGreaterThan(minFPS);
-		});
-	});
-
-	it('should have a good FID and CLS', async () => {
-		await page.evaluateOnNewDocument(FID);
-		await page.evaluateOnNewDocument(CLS);
-		await page.goto('http://localhost:8080/virtualList');
-		await page.waitForSelector('#virtualList');
-		await page.focus('#virtualList');
-		await page.keyboard.down('Enter');
-
-		let actualFirstInput = await firstInputValue();
-		let actualCLS = await clsValue();
-
-		TestResults.addResult({component: component, type: 'FID', actualValue: Math.round((actualFirstInput + Number.EPSILON) * 1000) / 1000});
-		TestResults.addResult({component: component, type: 'CLS', actualValue: Math.round((actualCLS + Number.EPSILON) * 1000) / 1000});
-
-		expect(actualFirstInput).toBeLessThan(maxFID);
-		expect(actualCLS).toBeLessThan(maxCLS);
-	});
-
-	it('should have a good DCL, FCP and LCP', async () => {
-		const filename = getFileName(component);
-
-		let passContDCL = 0;
-		let passContFCP = 0;
-		let passContLCP = 0;
-		let avgDCL = 0;
-		let avgFCP = 0;
-		let avgLCP = 0;
-		for (let step = 0; step < stepNumber; step++) {
-			const virtualListPage = await testMultiple.newPage();
-
-			await virtualListPage.tracing.start({path: filename, screenshots: false});
-			await virtualListPage.goto('http://localhost:8080/virtualList');
-			await virtualListPage.waitForSelector('#virtualList');
-			await virtualListPage.waitForTimeout(200);
-
-			await virtualListPage.tracing.stop();
-
-			const {actualDCL, actualFCP, actualLCP} = PageLoadingMetrics(filename);
-			avgDCL = avgDCL + actualDCL;
-			if (actualDCL < maxDCL) {
-				passContDCL += 1;
-			}
-
-			avgFCP = avgFCP + actualFCP;
-			if (actualFCP < maxFCP) {
-				passContFCP += 1;
-			}
-
-			avgLCP = avgLCP + actualLCP;
-			if (actualLCP < maxLCP) {
-				passContLCP += 1;
-			}
-
-			await virtualListPage.close();
-		}
-		avgDCL = avgDCL / stepNumber;
-		avgFCP = avgFCP / stepNumber;
-		avgLCP = avgLCP / stepNumber;
-
-		TestResults.addResult({component: component, type: 'DCL', actualValue: Math.round((avgDCL + Number.EPSILON) * 1000) / 1000});
-		TestResults.addResult({component: component, type: 'FCP', actualValue: Math.round((avgFCP + Number.EPSILON) * 1000) / 1000});
-		TestResults.addResult({component: component, type: 'LCP', actualValue: Math.round((avgLCP + Number.EPSILON) * 1000) / 1000});
-
-		expect(passContDCL).toBeGreaterThan(passRatio * stepNumber);
-		expect(avgDCL).toBeLessThan(maxDCL);
-
-		expect(passContFCP).toBeGreaterThan(passRatio * stepNumber);
-		expect(avgFCP).toBeLessThan(maxFCP);
-
-		expect(passContLCP).toBeGreaterThan(passRatio * stepNumber);
-		expect(avgLCP).toBeLessThan(maxLCP);
-	});
-});
+listItemTests(componentName, 12);
+listItemTests(componentName, 100);
```

Comment:
I think to keep the previous result file name (VirtualList.txt. not VirtualList100.txt), I should be
```suggestion
listItemTests(componentName);
```

Am I right?